garak/_config.py

```diff
@@ -72,6 +72,15 @@ class TransientConfig(GarakSubConfig):
     starttime = None
     starttime_iso = None
 
+    # Resume mode fields
+    resume_file = None  # Path to JSONL report to resume from
+    completed_attempts = (
+        None  # Dict of {probe_classname: set(seq)} for completed attempts
+    )
+    pending_detection_attempts = (
+        None  # Dict of {probe_classname: {seq: attempt_data}} for status=1 attempts
+    )
+
```

Comment:
`_config` is not meant as a storage location, this is not the correct place to track this.